configs/BKMN-NERO.config

```diff
@@ -1,68 +1,79 @@
-# Betaflight / STM32F7X2 (S7X2) 4.0.0
+# Betaflight / STM32F7X2 (S7X2) 4.2.0 May  5 2020 / 15:49:43 (b480103) MSP API: 1.43
```

Comment:
This config is for Rotorflight, so please add the relevant line here which firmware you used for testing this.